packages/itwin/reports-config-widget/src/widget/components/Reports.tsx

```diff
@@ -176,7 +186,7 @@ export const Reports = () => {
                 onClick={updateDatasets}
                 disabled={selectedReportIds.length === 0}
               >
-                <SvgRefresh />
+                < SvgPlay />
```

Comment:
Formatting - remove spaces after the opening `<`.

Reply:
Space removed